src/session/session_api.c

```diff
@@ -2120,6 +2120,8 @@ __wt_open_internal_session(WT_CONNECTION_IMPL *conn, const char *name, bool open
      */
     F_SET(session, session_flags | WT_SESSION_INTERNAL);
 
+    /* Internal sessions default to read-committed isolation. */
+    session->isolation = session->txn->isolation = WT_ISO_READ_COMMITTED;
```

Comment:
Ah - this is one of the reasons why this change is smaller than I'd expected. I think it's clever - did you try going down the other path and seeing which internal operations need read-committed versus just happen to be using it?

Reply:
I changed it for only one internal session - meta ckpt session and ran a WT patch build, found no issues.